### PR TITLE
Fix for layouts in a cell to expand on resize (WPF) Issue #5682

### PR DESCRIPTION
**Pull request by jaytilly**
### Description of Change ###
The child layouts where not receiving any notification to re-layout.  This fix calls layout on the children view items.

### Issues Resolved ### 
Grid will now auto expand within a ViewCell

### PR Checklist ###
- [] Has automated tests (Visual Tested)
- [x] Rebased on top of the target branch at time of PR (Not needed)
- [x] Changes adhere to coding standard

### CODE REVIEW

File: Xamarin.Forms.Platform.WPF/CellControl.cs [hunk 1]
```diff
@@ -85,6 +85,27 @@ void SetSource(object oldCellObj, object newCellObj)
...
+			if (Content is ViewCell vc)
+			{
+				if (vc.LogicalChildren != null && vc.LogicalChildren.Any())
+				{
+					foreach (var child in vc.LogicalChildren)
+					{
+						if (child is Layout<View> layout)
+						{
+							if (layout.HorizontalOptions.Expands)
+							{
```
Comment (rmarinho) [inline Xamarin.Forms.Platform.WPF/CellControl.cs]: Why only if this ? we should layout always  no ?

Comment (jaytilly) [reply Xamarin.Forms.Platform.WPF/CellControl.cs]: Probably - I should have seen that.  I can fix and test this evening. 

Thanks for the review

Comment (rmarinho) [reply Xamarin.Forms.Platform.WPF/CellControl.cs]: i still don't get why we just call layout if `layout.HorizontalOptions.Expands` ?

Comment (jaytilly) [reply Xamarin.Forms.Platform.WPF/CellControl.cs]: Good question.. my assumptions may not be valid. Should the child layout of a cell always expand to fit the cell?

Comment (VladislavAntonyuk) [reply Xamarin.Forms.Platform.WPF/CellControl.cs]: We have to always layout content. I have fixed it in my PR.

**Pull request merged**